src/vs/editor/contrib/peekView/browser/peekView.ts

```diff
@@ -53,7 +53,7 @@ registerSingleton(IPeekViewService, class implements IPeekViewService {
 		};
 		this._widgets.set(editor, { widget, listener: widget.onDidClose(remove) });
 	}
-});
+}, true);
```

Comment:
Intentional `true`?

Reply:
Yeah, that's was an adoption to make it support delayed already